target/linux/ramips/dts/mt7628an_hiwifi_hc5761a.dts

```diff
@@ -32,12 +32,23 @@
 			linux,default-trigger = "phy1tpt";
 		};
 	};
+
+	gpio_export {
+		compatible = "gpio-export";
+		#size-cells = <0>;
+
+		usbpower {
+			gpio-export,name = "usb";
```

Comment:
Shouldn't tthis be "hiwifi:power:usb"?

Reply:
I have test on hc5761a,It works Okay。please merge